neuromation/cli/project.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from cookiecutter.main import cookiecutter
+
+from .root import Root
+from .utils import async_cmd, command, group
+
+
+log = logging.getLogger(__name__)
+
+
+@group()
+def project() -> None:
+    """
+    Project operations.
+    """
+
+
+@command()
+@async_cmd()
+async def init(root: Root) -> None:
+    """
+    Initialize an empty project.
+
+    Examples:
+
+    # Initializes a scaffolding for the new project with the recommended
+    # project structure:
+    #   .
+    #   ├── {project_name}     # directory with your code
```

Comment:
Please drop the project structure tree.
Otherwise, you must update the docstring after every cookiecutter template change, I doubt if it is realistically possible.

Reply:
Hmm.. As far as I know, cookiecutter project structure is pretty much stable (changes cause lots of problems since we will build our funcitonality basing on it). Personally, I find project structure in docstring very informative.

cc @mariyadavydova

Reply:
Please just drop it. Period.

Reply:
This structure may change several times until we reach some stable point. As this command is separated from the template itself, we will most probably forget to update this docstring.

Let's drop it and add a link to the template repo instead (which is perfectly valid, we are using an established open-source tool and can tell about that).

Reply:
OK